### PR TITLE
Add issue templates

### PR DESCRIPTION
**Pull request by silkeh**
Add templates for bugs, package requests and package update requests based on the templates on Phabricator.

- [x] Requesting a new package
- [x] Reporting a bug
- [x] Requesting a package update

### CODE REVIEW

**Review by TraceyC77 — DISMISSED** [review]
Looks good to me.

File: .github/ISSUE_TEMPLATE/request-package-update.yaml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+      description: Name of the software.
+    validations:
+      required: true
+  - type: input
+    id: version
+    attributes:
+      label: Version
+      description: Requested version.
+    validations:
+      required: true
```
Comment (EbonJaeger) [inline .github/ISSUE_TEMPLATE/request-package-update.yaml:20]: When the package is updated, it should be updated to the latest stable version anyway, barring build issues, issues with dependencies or reverse dependencies, etc. Knowing that, is this field really helpful?

Comment (silkeh) [reply .github/ISSUE_TEMPLATE/request-package-update.yaml:20]: People might request a specific version for specific reasons, so I'd like to keep the field. Definitely shouldn't be required though.

Comment (EbonJaeger) [reply .github/ISSUE_TEMPLATE/request-package-update.yaml]: Good enough reason for me.

**Review by EbonJaeger — APPROVED** [review]
LGTM, thanks! :)

**Comment by joebonrichie** [comment]
LGTM.

Note to self: update https://github.com/solus-packages/.github/blob/main/profile/README.md when merged

**Pull request merged**